importer/src/powdr_std.rs

```diff
@@ -40,8 +40,12 @@ fn load_std<T: FieldElement>() -> ASMModule<T> {
         }
         Ok(std_source) => {
             let std_content =
-                parse_asm::<T>(Some(std_path.as_path().to_str().unwrap()), &std_source).unwrap();
-
+                parse_asm::<T>(Some(std_path.as_path().to_str().unwrap()), &std_source)
```

Comment:
I have no idea why this calls `parse_asm` directly instead of calling the module loader.